### ISSUE TITLE
ModalTrigger: no backward in minor version changes

### ISSUE THREAD

**Issue by doombu** [react-bootstrap/react-bootstrap#929]
You removed ModalTrigger, now all projects with this component are broken. And all this in minor version changes.

**Comment by mathieumg** [comment]
Are you unable to use it or is it just by reading the changelog? It still exists, it's only deprecated: https://github.com/react-bootstrap/react-bootstrap/blob/master/src/ModalTrigger.js#L117-L118

**Comment by doombu** [comment]
@mathieumg thank you for fast answer.
With version 0.23.6 I have deprecated message, then have error with google maps lib:
"Uncaught TypeError: Cannot read property 'offsetWidth' of null"
(gmaps I use in this modal window)

With 0.23.5 I have no errors.

**Comment by mathieumg** [comment]
That's definitely not intended, we'll investigate! Thank you.

**Comment by doombu** [comment]
more details. I have GoogleMapsModal component.

```javascript
render() {
    return (
            <Modal {...this.props} className='google-maps-modal' title='Modal heading' animation={true}>
                <div className="modal-body">
                    <div id="map-canvas"></div>
                </div>
            </Modal>
    );
}

componentDidMount() {
    ....
    this.map = new google.maps.Map(document.getElementById('map-canvas'), mapOptions);
    ....
}
```

and in 0.23.6 document.getElementById('map-canvas') is null, thats why I have offsetWidth error then.

**Comment by doombu** [comment]
one more.

at 0.23.5 the code from my previous comment executes only after ModalTrigger clicked:
```javascript
            <ModalTrigger modal={<GoogleMapsModal {...this.props}/>}>
                <div className="last-mile-assign-view-map">
                    {this.props.linkText}
                </div>
            </ModalTrigger>
```

but at 0.23.6 the code executes immediately.

**Comment by taion** [comment]
I just released v0.23.7, which fixes this.

**Comment by doombu** [comment]
thx, works fine!
